src/components/LogoutButton/LogoutButtonComponent.js

```diff
@@ -1,15 +1,23 @@
-import React, { Component } from 'react';
-
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
 import Button from '@material-ui/core/Button';
+import { colors } from 'colors';
+
+const useStyles = makeStyles({
+  logoutButton: {
```

Comment:
I believe these styles are not specific to the `logoutButton`.
All dropdown menus in the app have the same style:
![image](https://user-images.githubusercontent.com/6686679/115116526-9f3c9c80-9fa2-11eb-96fa-4b7ca5b541ea.png)

![image](https://user-images.githubusercontent.com/6686679/115116557-b7acb700-9fa2-11eb-8df1-6fd42c650bea.png)

![image](https://user-images.githubusercontent.com/6686679/115116540-a9f73180-9fa2-11eb-9e0a-a2d67bfb1423.png)

So I think we should apply this style to the MUI `MenuItem` in the [theme.js](https://github.com/DarthVanger/NeuralNotes/blob/master/src/theme.js#L81). We already have some styles for the MenuItem there:
```
    MuiMenuItem: {
      root: {
        background: colors.dialogsGray,
        minHeight: '32px',
      },
    },
```

Reply:
I used styles ```<Typography>``` from the theme, but in order to remove ```textTransform: 'capitalize'```, instead of a button, we need to use another tag, not a ```<button>```.